### ISSUE TITLE
`property-no-unknown` false positive for descriptor `navigation`

### ISSUE THREAD

**Issue by sebdanielsson** [stylelint/stylelint#7751]
### What minimal example or steps are needed to reproduce the bug?

```css
@view-transition {
  navigation: auto;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "property-no-unknown": true
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css"
```

### Which Stylelint-related dependencies are you using?

```json
{
    "stylelint": "^16.6.1",
    "stylelint-config-html": "^1.1.0",
    "stylelint-config-standard": "^36.0.0",
}
```

### What did you expect to happen?

Lint should pass.

### What actually happened?

```
20:3  ✖  Unexpected unknown at-rule "@view-transition"  at-rule-no-unknown
21:5  ✖  Unexpected unknown property "navigation"       property-no-unknown
```

**Comment by Mouvedia** [comment]
It would be better for the web community if it was solved on [known-css-properties](https://github.com/known-css/known-css-properties) side.

**Comment by DATEx2** [comment]
Not sure if it is related, but this still occurs in VSCode 
```
Version: 1.96.4 (user setup)
Commit: cd4ee3b1c348a13bafd8f9ad8060705f6d4b9cba
Date: 2025-01-16T00:16:19.038Z
Electron: 32.2.6
ElectronBuildId: 10629634
Chromium: 128.0.6613.186
Node.js: 20.18.1
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.19045
```
